README.md

```diff
@@ -61,3 +98,16 @@ work to the project under the project's open source license. Whether or not you
 explicitly, by submitting any copyrighted material via pull request, email, or other means you agree
 to license the material under the project's open source license and warrant that you have the legal
 authority to do so.
+For more detailed step-by-step tutorials, please read 
+[Contribute to Alluxio](https://www.alluxio.org/docs/1.8/en/contributor/Contributor-Getting-Started.html). For [new contributor tasks](https://github.com/Alluxio/new-contributor-tasks), please limit 2 tasks per new contributor.
+
+## Useful Links
+
+- [Alluxio Open Source Website](http://www.alluxio.org/) 
+- [Alluxio Inc.](http://www.alluxio.com/)
+- [Alluxio Latest Release Document](https://www.alluxio.org/download/releases/)
+- [Master Branch Document](https://www.alluxio.org/docs/master/en/)
```

Comment:
I'd remove this since latest release is pretty much always better.

Reply:
done